json_serialization_writer_test.go

```diff
@@ -303,59 +306,87 @@ func TestEscapeTabAndCarriageReturnInStrings(t *testing.T) {
 // properly encoded as JSON.
 func TestShortEscapeSequencesInString(t *testing.T) {
 	// Expected results for each test are quoted since it's a JSON string.
+	skipTestCheck := func() bool {
```

Comment:
why don't we upgrade our CI from go1.19 to 1.22 instead?
(not suggesting to upgrade the package go version)